train.py

```diff
@@ -91,11 +118,13 @@
     alignment=config["alignment"],
 )
 
-if config["pos"] == "both":
+if config["pos"] == "both_34":
     label_names_258, reader_258 = load_reader(pos="34S_19E_258N", **kwargs)
     print("\u2713 Loaded 258")
     label_names_259, reader_259 = load_reader(pos="34S_19E_259N", **kwargs)
     print("\u2713 Loaded 259")
+    reader_258.labels.reset_index(inplace=True)
+    reader_259.labels.reset_index(inplace=True)
```

Comment:
1. Why not reset the index after concatenation?
2. Why does it work for South Africa data as is, but this change is needed for the Germany data?

Reply:
2. < Give me a second >

Reply:
This makes sense